openlibrary/plugins/upstream/code.py

```diff
@@ -27,12 +27,13 @@
 import merge_authors
 
 if not config.get('coverstore_url'):
-    config.coverstore_url = "http://covers.openlibrary.org"
+    config.coverstore_url = "https://covers.openlibrary.org"
 
 class static(delegate.page):
     path = "/images/.*"
     def GET(self):
-        raise web.seeother('/static/' + web.ctx.path)
+        host = 'https://%s' % web.ctx.host if 'openlibrary.org' in web.ctx.host else ''
```

Comment:
@mekarpeles: What's your motivation behind explicitly coding in the HTTPS host? Will not the relative path `/static/` already use the existing protocol (which should be HTTPS)?